src/inputs/Switch.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <label :class="switchClass">
+        <span>{{ this.label }}</span>
+        <div 
+            :name="name"
+            v-on:click="e => this.onChange(!inputValue)"
+        />
+        <field-error :errors="this.validationErrors" />
+    </label>
+</template>
+
+<script>
+    import Vue from 'vue';
+    import _ from 'lodash';
+    import inputProps from '../inputProps';
+    import { validationMixin } from '../validationHelper';
+
+    export default {
+        name: 'SuperSwitch',
+        mixins: [validationMixin],
+        props: inputProps,
+        data() {
+            return {
+                inputValue: this.value,
+            }
+        },
+        computed: {
+            defaultValue() {
+                return false;
+            },
+            switchClass() {           
+                return this.value ?
+                    'form__switch' : 'form__switch form__switch--off';
+            }
+        },
+        methods: {
+            onChange(value) {
+                this.inputValue = value;
+                this.$emit('onChange', this.name, value);
+            }
+        }
+    }
```

Comment:
when adding `formSchemaMixin` to this component, remove all obsolete properties.